pkg/v3/outcome.go

```diff
@@ -247,17 +247,17 @@ func (bo *BasicOutcome) UnmarshalJSON(b []byte) error {
 	}
 
 	metadata := make(map[OutcomeMetadataKey]interface{})
-	for key, value := range rawOutcome.Metadata {
+	for key := range rawOutcome.Metadata {
 		switch OutcomeMetadataKey(key) {
 		case CoordinatedBlockOutcomeKey:
 			// value is a block history type
-			var bk ocr2keepers.BlockKey
+			// var bk ocr2keepers.BlockKey
 
-			if err := json.Unmarshal(value, &bk); err != nil {
-				return err
-			}
+			// if err := json.Unmarshal(value, &bk); err != nil {
+			// 	return err
+			// }
 
-			metadata[CoordinatedBlockOutcomeKey] = bk
+			// metadata[CoordinatedBlockOutcomeKey] = bk
```

Comment:
Same as observation, commenting out in this PR as this breaks with new types. Not doing a proper fix as the overall logic needs to be rethought